tutoraspects/templates/openedx-assets/assets/charts/Distinct_forum_users.yaml

```diff
@@ -0,0 +1,71 @@
+_file_name: Distinct_forum_users.yaml
+cache_timeout: null
+certification_details: null
+certified_by: null
+dataset_uuid: 39d1e786-c0c8-4c56-81c8-56fb0df88001
+description: null
+params:
+  adhoc_filters: []
+  dashboards:
+  - 61
+  datasource: 57__table
+  extra_form_data: {}
+  granularity_sqla: emission_time
+  header_font_size: 0.4
+  metric:
+    aggregate: COUNT_DISTINCT
+    column:
+      advanced_data_type: null
+      certification_details: null
+      certified_by: null
+      column_name: actor_id
+      description: null
+      expression: null
+      filterable: true
+      groupby: true
+      id: 445
+      is_certified: false
+      is_dttm: false
+      python_date_format: null
+      type: String
+      type_generic: 1
+      verbose_name: null
+      warning_markdown: null
+    datasourceWarning: false
+    expressionType: SIMPLE
+    hasCustomLabel: false
+    label: COUNT_DISTINCT(actor_id)
+    optionName: metric_5y4uvwa13v4_f12i3twecs6
+    sqlExpression: null
+  subheader_font_size: 0.15
+  time_format: smart_date
+  time_range: No filter
+  viz_type: big_number_total
+  y_axis_format: SMART_NUMBER
+query_context: '{"datasource": {"id": 57, "type": "table"}, "force": false, "queries":
```

Comment:
Can you remove these `query_context`s? The size of the assets is big enough that it might push us over the limit in k8s again, so anything we can do to keep these smaller is helpful.

Reply:
We should add a clean step in the serialize command to automatically remove not needed assets fields